index.js

```diff
@@ -1,88 +1,129 @@
 // ***** Question 1 *****
 // *** Uncomment the lines below to test
-// console.log("%cQuestion 1", "color: red") 
+console.log("%cQuestion 1", "color: red") 
 
-// console.log(counter) 
-// // => 2
-// console.log("%c----------", "color: red") 
+// solution:
+let counter = 1
+counter = 2
+console.log(counter) 
+// 
+
+console.log("%c----------", "color: red") 
 
 
 // ***** Question 2 *****
 // *** Uncomment the lines below to test
 // *** after testing, comment the line causing the error back in 
 // *** otherwise, the error will stop the rest of your code from running
-// console.log("%cQuestion 2", "color: red") 
+console.log("%cQuestion 2", "color: red") 
 
-// console.log(name)
-// => "Raffy"
+// solution
+const name = "Noura"
+console.log(name)
 // name = "Not Raffy" 
-// => TypeError
-// console.log("%c----------", "color: red") 
+// 
```

Comment:
NIce job using "let" and "const" in the correct context to define your variable here.